javascriptv3/example_code/cross-services/video-analyzer/README.md

```diff
@@ -0,0 +1,27 @@
+# TypeScript environment for the Video Analyzer App tutorial
+
+This is a workspace where you can find working AWS SDK for JavaScript version 3 (v3) 'Video analyzer app' tutorial.
+
+The [AWS SDK for JavaScript v3 Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/video-analyzer.html) contains this examples.
+
+# Getting started
```

Comment:
Seems like a lot of basic information about both configuring the SDK and writing JavaScript code.

We do not teach programming. I would just add a blurb like:

If you are not familiar with setting up and configuring the AWS SDK for JavaScript, see <link to JS dev guide's "Getting started" topic>.

Reply:
Its only two steps to explain how to setup a project environment to run the examples. I think its useful, especially for users who link here  the Prerequisites section in the documentation - [see example](
![image](https://user-images.githubusercontent.com/40031786/117153227-ae00bd00-adb2-11eb-9691-f6d98349e2e1.png) 
Also, there is an equivalent page for each set of service examples. So I'll take a look at them in light of your comments as we implement Scouring of the Services, and implement the same change across all if necessary. 
) his is where users land from the documentation for these examples. The idea is to give them the bare bones for setting up a project environment to run the examples.